tests/test_http_ext_auth.py

```diff
@@ -315,6 +317,21 @@ class TestHttpExtAuth(tb.ExtAuthTestCase):
         CONFIGURE CURRENT DATABASE SET
         ext::auth::AuthConfig::token_time_to_live := <duration>'24 hours';
 
+        CONFIGURE CURRENT DATABASE SET
+        ext::auth::SMTPConfig::sender := 'noreply@example.com';
+
+        CONFIGURE CURRENT DATABASE SET
+        ext::auth::SMTPConfig::timeout_per_attempt := <duration>'1 seconds';
+
+        CONFIGURE CURRENT DATABASE SET
+        ext::auth::SMTPConfig::timeout_per_email := <duration>'1 seconds';
+
+        CONFIGURE CURRENT DATABASE SET
+        ext::auth::SMTPConfig::validate_certs := true;
+
+        CONFIGURE CURRENT DATABASE SET
+        ext::auth::SMTPConfig::security := ext::auth::SMTPSecurity.TLS;
```

Comment:
I was expecting the default values to kick in without setting these manually, but it turned out not the case.

Reply:
Hm, this is odd. @msullivan, any idea why?

Reply:
This is probably because we didn't use the `spec.Setting.default` value in `auth/utils.py`, I'll push a fix.